dms/__manifest__.py

```diff
@@ -9,7 +9,13 @@
     "license": "LGPL-3",
     "website": "http://github.com/OCA/dms",
     "author": "MuK IT,Odoo Community Association (OCA)",
-    "depends": ["web_drop_target", "mail", "web_view_searchpanel", "http_routing"],
+    "depends": [
+        "web_drop_target",
+        "mail",
+        "web_view_searchpanel",
+        "http_routing",
+        "mail_preview_base",
```

Comment:
Why adding it as dependency if not technically needed? You are already stating that it's recommended to install, but it shouldn't be mandatory.

Reply:
The base is required in order to use the `preview_binary` widget

![imagen](https://user-images.githubusercontent.com/28590170/94422843-fb1b1180-0187-11eb-823e-a7568ba9d0c6.png)

I have changed the readme in order to improve the explanation

Reply:
But I think we shouldn't add a new dependency in the middle of the release...

Reply:
OK, I see that you are removing some JS code here. Uhm, difficult balance to add or not to add... Question, with current JS code, mail_preview_* work? Does this code interfere with `mail_preview_base`?

Reply:
The `mail_preview_base` defines a base of functions that will be used to simplify how to preview files.

It also defines a new widget that will allow to preview the file if we can preview it (preview_binary). This widget is used on DMS and the logic will also be reused on `dms_field` module.

So, base is required in order to make this widget reusable, keeping the widget on `dms` or in `mail_preview_base` is optional, but I preferred to keep it all on the same place.

Reply:
OK, let's go with it